ecosystem/typescript/sdk_v2/src/types/index.ts

```diff
@@ -84,6 +94,451 @@ export type MoveModuleBytecode = {
   abi?: MoveModule;
 };
 
+/////// TRANSACTION TYPES ///////
+
+export type Transaction =
+  | PendingTransaction
+  | UserTransaction
+  | GenesisTransaction
+  | BlockMetadataTransaction
+  | StateCheckpointTransaction;
```

Comment:
```suggestion
export type TransactionResponse =
  | PendingTransactionResponse
  | UserTransactionResponse
  | GenesisTransactionResponse
  | BlockMetadataTransactionResponse
  | StateCheckpointTransactionResponse;
```

commenting as a reminder of our earlier conversation, not necessary but may make things a little clearer